README.md

```diff
@@ -1,5 +1,63 @@
+![🚧 Under construction 👷‍♂️](https://i.imgur.com/LEP2R3N.png)
+
+# Community devcontainer templates
+
+📋 Community-driven collection of templates for [@devcontainers]
+
 <div align="center">
 
-![🚧 Under construction 👷‍♂️](https://i.imgur.com/LEP2R3N.png)
+![](https://i.imgur.com/Us84cLP.png)
 
 </div>
+
+🔧 80% of the configuration you'll ever need \
+💻 Ready to go with GitHub Codespaces \
+🚀 Quickly get up-and-running with a devcontainer
+
+**🤝 We want you!** If you're a community devcontainer template maintainer, we'd
+love to add you to this project! ❤️ If you just want to hand off a devcontainer
+template that you no longer want to maintain, **that's perfect too**. Open a
+[Discussion] and let's have a friendly chat. 😊
```

Comment:
@eitsupi it's been three weeks 🙄 ... does this fit with kinda your idea of what i said:

> I'm thinking a **bold banner** of text at the top for the "Encourage existing community members who have possible devcontainer templates to reach out"

cause i honestly have forgotten what i meant by that or what was even going on 3 weeks ago. my bad 😅

Reply:
I think this is great.

My concern is that there are numerous community templates left on https://github.com/microsoft/vscode-dev-containers that are no longer updated already.
I think it would be great if we could continue to update them somewhere else.